spaceship.zsh

```diff
@@ -446,11 +448,20 @@ spaceship_host() {
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  _prompt_section \
-    "$SPACESHIP_DIR_COLOR" \
-    "$SPACESHIP_DIR_PREFIX" \
-    "%${SPACESHIP_DIR_TRUNC}~" \
-    "$SPACESHIP_DIR_SUFFIX"
+  _is_git
+  if [ $? -eq 0 ] && [ $SPACESHIP_DIR_TRUNC_IN_GIT == true ]; then
+    _prompt_section \
+      "$SPACESHIP_DIR_COLOR" \
+      "$SPACESHIP_DIR_PREFIX" \
+      "${PWD##*/}" \
```

Comment:
We could fetch the root of git repository using `git rev-parse --show-toplevel` and path it to by `git rev-parse --show-cdup`. Can we use this or other methods to get the path from git root ?